src/Service/UrlBundleService.php

```diff
@@ -18,25 +19,19 @@ class UrlBundleService implements UrlBundleInterface
 {
     use Extensible;
     use Injectable;
+    use Configurable;
 
-    /**
-     * @var array
-     */
-    protected $urls = [];
+    private static bool $strip_stage_param = true;
+
+    protected array $urls = [];
```

Comment:
Can't add a type to a protected property as it's technically an API change
```suggestion
    protected $urls = [];
```